qutebrowser/javascript/webengine_caret.js

```diff
@@ -0,0 +1,1401 @@
+/* eslint-disable max-lines, max-len, max-statements, complexity,
```

Comment:
If you don't mind, can you please add yourself and this file to `.github/CODEOWNERS`? That way, GitHub will automatically request a review from you if somebody modifies this file in a PR.

Reply:
Also, I think this file should be renamed to just `caret.js` for consistency with the other files (most of them are QtWebEngine-specific anyways). Please make sure to do so in a separate commit though.